src/library/Uno.Material/Styles/Controls/TextBox.xaml

```diff
@@ -42,7 +48,7 @@
 									<VisualState.Setters>
 										<Setter Target="ContentElement.Opacity"
 												Value="{StaticResource ControlPressedOpacity}" />
-										<Setter Target="PlaceholderTextElement.Opacity"
+										<Setter Target="PlaceholderElement.Opacity"
 												Value="{StaticResource ControlPressedOpacity}" />
```

Comment:
Is ControlPressedOpacity a system value?

Reply:
It's an old value that should be removed altogether from all controls.